tests/framework/installer/ceph_installer.go

```diff
@@ -534,8 +534,12 @@ func (h *CephInstaller) UninstallRookFromMultipleNS(systemNamespace string, name
 		"objectbuckets.objectbucket.io",
 		"objectbucketclaims.objectbucket.io",
 		"cephrbdmirrors.ceph.rook.io")
-	checkError(h.T(), err, "cannot delete CRDs")
 
+	checkError(h.T(), err, "cannot delete CRDs")
+	logger.Infof("--------Now wil check if CRD are deleted or not------")
+	if err != nil {
+		logger.Infof("----Cannot delete CRDs-----")
```

Comment:
Some suggestions to try when `err != nil`:
- kubectl get cephcluster <name> -o yaml // see if there are existing cephcluster CRs with finalizers that are preventing the deletion of the CRD
- If the cephcluster CR still exists, force remove the finalizer. Retry this several times if it keeps failing
- Wait and retry the DeleteResource multiple times if it fails